NeuralAmpModeler/resources/fonts/Ronduit-Capitals-Light-License.txt

```diff
@@ -0,0 +1,41 @@
+Preamble
+In this license, 'the given typeface' refers to the given .zip file, which may contain one or numerous fonts. These fonts can be of any type (.ttf, .otf, ...) and together they form a 'font family' or in short a 'typeface'.
+
+1. Copyright
+The given typeface is the intellectual property of
+Ivan Filipov
+Neogrey Ltd
+Plovdiv, Bulgaria
+
+provided it is original, and is protected by copyright laws in many parts of the world.
+
+2. Personal Use
+The given typeface may be downloaded and used free of charge for personal use, as long as the usage is not racist or illegal. Personal use refers to all usage that does not generate financial income in a business manner, for instance:
+
+- personal scrapbooking for yourself
+- recreational websites and blogs for friends and family
+- prints such as flyers, posters, t-shirts for churches, charities, and non-profit organizations
+
+3. Commercial Use

```

Comment:
Oh dear...this is problematic--this license appears to conflict with the open-source license that this repo has.

Can I ask that this font be dropped from this PR?

Reply:
Thought I was doing the right thing including the license TBH.

Reply:
@olilarkin you were. Uncomfortable but good to address and I'm happier for the outcome. Thanks!